src/web3/contracts.ts

```diff
@@ -6,36 +6,70 @@
 import { Lock } from '@celo/base/lib/lock'
 import { ContractKit, newKitFromWeb3 } from '@celo/contractkit'
 import { sleep } from '@celo/utils/lib/async'
-import GethBridge from 'react-native-geth'
-import { call, delay, select } from 'redux-saga/effects'
+// import GethBridge from 'react-native-geth'
+import { call, select } from 'redux-saga/effects'
 import { ContractKitEvents } from 'src/analytics/Events'
 import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
 import { ErrorMessages } from 'src/app/ErrorMessages'
+import { CapsuleWallet } from 'src/capsule/react-native/ReactNativeCapsuleWallet'
 import { DEFAULT_FORNO_URL } from 'src/config'
-import { isProviderConnectionError } from 'src/geth/geth'
-import { GethNativeBridgeWallet } from 'src/geth/GethNativeBridgeWallet'
-import { waitForGethInitialized, waitForGethSync, waitForGethSyncAsync } from 'src/geth/saga'
+//import { isProviderConnectionError } from 'src/geth/geth'
+//import { GethNativeBridgeWallet } from 'src/geth/GethNativeBridgeWallet'
+import { waitForGethSync, waitForGethSyncAsync } from 'src/geth/saga'
 import { navigateToError } from 'src/navigator/NavigationService'
 import Logger from 'src/utils/Logger'
 import { getHttpProvider, getIpcProvider } from 'src/web3/providers'
 import { fornoSelector } from 'src/web3/selectors'
 import Web3 from 'web3'
+import { EIP712TypedData } from '@celo/utils/lib/sign-typed-data-utils'
+import { ReactNativeSignersStorage } from '../capsule/react-native/ReactNativeSignersStorage'
 
 const TAG = 'web3/contracts'
-const KIT_INIT_RETRY_DELAY = 2000
+// const KIT_INIT_RETRY_DELAY = 2000
 const CONTRACT_KIT_RETRIES = 3
 const WAIT_FOR_CONTRACT_KIT_RETRIES = 10
 
-let wallet: GethNativeBridgeWallet | undefined
+let wallet: CapsuleWallet | undefined // GethNativeBridgeWallet | undefined
 let contractKit: ContractKit | undefined
 
 const initContractKitLock = new Lock()
 
+// @ts-ignore
+async function signTestTransaction(newWallet: CapsuleWallet) {
+  Logger.debug('TEST SIGNING')
+  const PAYLOAD: EIP712TypedData = {
+    types: {
+      EIP712Domain: [
+        { name: 'name', type: 'string' },
+        { name: 'version', type: 'string' },
+        { name: 'chainId', type: 'uint256' },
+      ],
+      Message: [{ name: 'content', type: 'string' }],
+    },
+    domain: {
+      name: 'Valora',
+      version: '1',
+      chainId: 17,
+    },
+    message: {
+      content: 'valora auth message',
+    },
+    primaryType: 'Message',
+  }
+  const storage = new ReactNativeSignersStorage()
+  const accounts = await storage.getAccounts()
+  const address = accounts[0]
+  const signedTypedMessage = await newWallet.signTypedData(address, PAYLOAD)
+  Logger.debug(signedTypedMessage)
+}
+
 async function initWallet() {
   ValoraAnalytics.track(ContractKitEvents.init_contractkit_get_wallet_start)
-  const newWallet = new GethNativeBridgeWallet(GethBridge)
+  const newWallet = new CapsuleWallet()
```

Comment:
Why is this `CapsuleWallet` and not `ReactNativeCapsuleWallet`, since that is the implementation class? I happened to be looking for the function used to refresh the session.